packages/composites/savax-price/CHANGELOG.md

```diff
@@ -0,0 +1,257 @@
+# @chainlink/example-composite-adapter
+
+## 1.0.32
```

Comment:
This should be empty, it's a new adapter.

Reply:
I think that we should update the `new` command script to not automatically fill in this file

Reply:
do I just commit an empty file then?